### ISSUE TITLE
RuntimeError: Unable to load plugin on Ubuntu (Linux)

### ISSUE THREAD

**Issue by music-lover99** [DBraun/DawDreamer#114]
When I run ```soft_clipper = engine.make_plugin_processor("my_soft_clipper", 'path\to\Initial Clipper.dll')``` on my computer running Windows 10, it works. But when I run that same line on my computer running Ubuntu 20.04, I get an error message.

The error message:

```
error attempt to map invalid URI '/home/sebas/project/plugins/Initial Clipper.dll'
soft_clipper = engine.make_plugin_processor("my_soft_clipper", 'path\to\Initial Clipper.dll')
Runtime Error: Unable to load plugin
```

Why is this only working in Windows and not Linux? How can I fix this?

**Comment by DBraun** [comment]
There may be workarounds, but my first guess is that this `Initial Clipper.dll` is a Windows-only plugin. You have to find a Linux alternative such as LV2, but I admittedly haven't tried LV2 plugins for DawDreamer. They might work.

**Comment by DBraun** [comment]
Yes. I think LV2 support is enabled. Other formats such as LADSPA and ARA are disabled according to the projucer file.

**Comment by DBraun** [comment]
This seems like a good start https://lv2plug.in/pages/projects.html

**Comment by jorshi** [comment]
I'm running into this issue as well trying to load LV2 plugins on Ubuntu.

Example trying to load a couple LV2 synth plugins on google colab: https://colab.research.google.com/drive/1CXfO1N7XMLVMxVK3Tw-MivBi2RIbPrey?usp=sharing

**Comment by jtag05** [comment]
I'm also having issues with LV2 plugins on Ubuntu. I'm curious if there's been any progress on this since it was opened.
